lms/templates/ux/reference/instructor_dashboard/cohorts.html

```diff
@@ -0,0 +1,188 @@
+                </div>
+                <div class="message-actions">
+                    <button class="action-primary action-create"><i class="icon icon-plus"></i> Add a Cohort Group</button>
+                </div>
+              </div>
+
+              <!-- Message: Error -->
+              <div class="msg msg-error">
+                <h3 class="message-title">There's currently an error with your cohorts configuration within this course.</h3>
+                <div class="message-copy">
+                    <p>Error output (if any and near-human legible/comprehendable can be displayed here)</p>
+                </div>
+                <div class="message-actions">
+                    <a href="#" class="action-primary action-review">Review Configuration in Studio</a>
+                </div>
+              </div>
+
+              <!-- Managing cohorts -->
+              <section class="cohort-management-settings has-tabs">
+                <h2 class="group-header-title"><span class="title-value">Cohort Name Can be Placed Here and Should Accommodate Almost Everything</span> <span class="group-count">(contains 12,546 students)</span></h2>
+                <div class="cohort-management-group-setup">
+                  <p class="cohort-management-group-text">Students are added to this group only when you provide their email addresses or usernames on this page.<br /><a href="#">What does this mean?</a></p>
+                  <div class="cohort-management-group-actions">
+                    <span class="float-right"><a href="#">Edit settings in Studio</a></a>
+                  </div>
+                </div>
+                <ul class="wrapper-tabs">
+                  <li class="tab"><a href="#">Manage Students in Cohort Groups</a></li>
+                  <li class="tab is-selected"><a href="#">Cohort Group Settings</a></li>
+                </ul>
+
+                <div class="cohort-management-details">
+                  <form action="" method="post" name="" id="cohort-management-details-form" class="cohort-management-details-form">
+                    <h3 class="form-title sr">Cohort Group Details</h3>
+                    <div class="form-fields">
+                      <div class="cohort-management-details-name field field-text">
+                        <h4 class="cohort-section-header no-top-margin">Cohort Group Name:</h4>
+                        <label for="cohort-details-name" class="label sr">Cohort Group Name</label>
+                        <input type="text" name="cohort-details-name" value="Cohort Name is Placed Here and Should Accommodate Almost Everything" class="input cohort-details-name" id="cohort-details-name" placeholder="Enter Your Cohort's Name" required />
+                      </div>
+                    </div>
+
+                    <h4 class="cohort-section-header divider-lv2">Students in this cohort group are:</h4>
```

Comment:
To me, this calls for a ```<fieldset>/<legend>``` rather than an ```<h4>```  Navigation aids like Headings are not designed to be used in forms unless it is a *very* large form or a form in which only a portion of it is expected to be filled out.   A screen reader user enters "form processing mode" and doesn't even encounter Headings, which is why its important to use standard form controls like ```labels``` and ```fieldsets``` to identify and demarcate form controls.